src/main/java/org/opensearch/security/user/UserFilterType.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.user;
+
+public enum UserFilterType {
+
+    ANY("any"),
+    INTERNAL("internal"),
+    SERVICE("service");
+
+    private String name;
+
+    UserFilterType(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
```

Comment:
Nit; Looks like this getter is not use by any of the code, lets remove it to keep the codebase / change smaller

Reply:
Nit: optional if you'd like, feel free to close without resolving if you'd prefer not.